static/app/views/performance/database/useDenylistedProjects.tsx

```diff
@@ -0,0 +1,36 @@
+import useProjects from 'sentry/utils/useProjects';
+
+interface Options {
+  enabled?: boolean;
+  projectId?: string[];
+}
+
+/**
+ * Returns a list of projects that are not eligible for span metrics
+ * because they were denylisted.
+ *
+ * @param options Additional options
+ * @param options.projectId List of project IDs to check against. If omitted, checks all organization projects
+ * @returns List of projects
+ */
+export function useDenylistedProjects(options?: Options) {
+  const {projects, fetching} = useProjects();
+
+  const shouldCheckAllProjects =
+    options?.projectId?.length === 0 || options?.projectId?.includes('-1');
+
+  const projectsToCheck = shouldCheckAllProjects
+    ? projects
+    : projects.filter(project => options?.projectId?.includes(project.id.toString()));
+
+  const denylistedProjects = projectsToCheck
+    .filter(project => {
+      return !project?.features.includes('span-metrics-extraction');
```

Comment:
is features really conditionally defined on `project`? This feels like something we should fix with the whole codebase tbh.

Reply:
It is not, removed